File: kimhyunji/PGS/Level 1/과일 장수.java
import java.util.Arrays;
class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        int length = score.length;
        
        Arrays.sort(score); //오름차순
        
        for(int i=length-m; i>=0; i-=m){
            answer+=(score[i]*m);
        }
        return answer;
    }
}

File: baekyeonsu/PGS/Lv. 1/과일 장수.js
function solution(k, m, score) {
  score.sort((a, b) => {
    return b - a;
  });
  let answer = 0;
  for (let i = m - 1; i < score.length; i += m) {
    if (score[i] > k) answer += k * m;
    else answer += score[i] * m;
  }
  return answer;
}
